config/config.go

```diff
@@ -13,6 +13,12 @@ var (
 	BlockchainInterval  = 3000 * time.Millisecond
 	SystemStatsInterval = 500 * time.Millisecond
 	WidgetInterval      = 1000 * time.Millisecond
+	LinechartInterval   = 5000 * time.Millisecond
```

Comment:
BarChart?

Reply:
for barchart its EarningRateInterval
Linechart we are not using yet